client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -201,12 +196,12 @@ const NoteComponent = (props: ControlProps) => {
         path,
         error
           ? undefined
-          : {
+          : ({
               text,
               authorId,
               authorName,
               created,
-            }
+            } as NoteSchema)
```

Comment:
`as NoteSchema` isn't necessary here.

Reply:
handleChange only takes an any, that change was just to keep sure this has the shape of a NoteSchema